filament/src/materials/MaterialHelpersShading.fxh

```diff
@@ -87,36 +166,44 @@ vec3 UnpackNormal(vec2 packedNormal, vec2 scale) {
     return vec3(x, y, sqrt(clamp(1.0 - x * x - y * y, 0.0, 1.0)));
 }
 
+vec2 swizzleIvec(vec3 x, ivec2 i) {
+    return vec2( x[i[0]], x[i[1]] );
+}
+
+vec3 swizzleIvec(vec3 x, ivec3 i) {
+    return vec3( x[i[0]], x[i[1]], x[i[2]] );
+}
+
 // This is a whiteout blended tripalanar normal mapping, where each plane's tangent frame is
```

Comment:
nit: Is this comment still up to date? We're using triplanar weights, but doing biplanar mapping.

Reply:
Good point, clarified it in https://github.com/shapr3d/filament/pull/84/commits/b684c2cf4c67033e25895b311c7ee87dd85b0b30 .